src/Application.php

```diff
@@ -0,0 +1,64 @@
+<?php declare(strict_types=1);
+
+namespace SymfonyDocsBuilder;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\Console\Application as BaseApplication;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+use SymfonyDocsBuilder\Command\BuildDocsCommand;
+
+class Application
+{
+    /** @var BaseApplication */
+    private $application;
+    /** @var BuildDocsCommand */
+    private $buildDocsCommand;
+
+    public function __construct(BaseApplication $application, BuildDocsCommand $buildDocsCommand)
```

Comment:
We should pass an array of commands here - as we’ll have many. I’m fine if we need to manually register these as an argument in the services file - being explicit (instead of adding more magic layers like a compiler pass) has big advantages of simplicity when working on a lib.